ROADMAP.md

```diff
@@ -19,33 +19,39 @@ The following is a list of prioritized roadmaps divided by OS (Linux/Windows) an
-| GCN5.1       | gfx906      |               |               |
-| GCN5         | gfx900      |               |               |
+| Architecture | LLVM target | Build Passing | Sanity Tested | Release Ready |
+| ------------ | ----------- | ------------- | ------------- | ------------- |
+| CDNA3        | gfx942      | ✅            | ✅            | ✅            |
+| **CDNA2**    | **gfx90a**  |               |               |               |
+| CDNA         | gfx908      |               |               |               |
+| GCN5.1       | gfx906      |               |               |               |
+| GCN5         | gfx900      |               |               |               |
 
 **AMD Radeon**
 
-| Architecture | LLVM target | Sanity Tested | Release Ready |
-| ------------ | ----------- | ------------- | ------------- |
-| **RDNA3**    | **gfx1100** | ✅            |               |
-| **RDNA3**    | **gfx1101** |               |               |
-| **RDNA3**    | **gfx1102** |               |               |
-| RDNA2        | gfx1030     |               |               |
-| GCN5.1       | gfx906      |               |               |
+| Architecture | LLVM target | Build Passing | Sanity Tested | Release Ready |
+| ------------ | ----------- | ------------- | ------------- | ------------- |
+| RDNA4        | gfx1201     | ✅            | ✅            | ✅            |
+| **RDNA3.5**  | **gfx1151** | ✅            |               |               |
+| **RDNA3**    | **gfx1102** | ✅            |               |               |
+| **RDNA3**    | **gfx1101** | ✅            |               |               |
+| **RDNA3**    | **gfx1100** | ✅            |               |               |
+| RDNA2        | gfx1030     |               |               |               |
+| RDNA2        | gfx1010     |               |               |               |
+| GCN5.1       | gfx906      |               |               |               |
 
 ### HIP Runtime and SDK on Windows
 
 Check [windows_support.md](docs/development/windows_support.md) on current status of development.
 
 **AMD Radeon**
 
-| Architecture | LLVM target | Sanity Tested | Release Ready |
-| ------------ | ----------- | ------------- | ------------- |
-| **RDNA3**    | **gfx1101** |               |               |
-| **RDNA3**    | **gfx1100** |               |               |
-| RDNA2        | gfx1030     |               |               |
-| GCN5.1       | gfx906      |               |               |
+| Architecture | LLVM target | Build Passing | Sanity Tested | Release Ready |
+| ------------ | ----------- | ------------- | ------------- | ------------- |
+| **RDNA4**    | **gfx1201** | ✅            |               |               |
+| **RDNA3.5**  | **gfx1151** | ✅            | ✅            |               |
+| **RDNA3**    | **gfx1101** | ✅            |               |               |
+| **RDNA3**    | **gfx1100** | ✅            |               |               |
```

Comment:
Could call gfx1100 on Windows sanity tested. That's what I develop on. I guess we don't yet have CI machines running _tests_ though.